app/livechat/client/views/app/livechatCurrentChats.js

```diff
@@ -312,16 +318,57 @@ Template.livechatCurrentChats.events({
 });
 
 Template.livechatCurrentChats.onCreated(async function() {
-	this.ready = new ReactiveVar(false);
-	this.limit = new ReactiveVar(20);
+	this.isLoading = new ReactiveVar(false);
+	this.offset = new ReactiveVar(0);
+	this.total = new ReactiveVar(0);
 	this.filter = new ReactiveVar({});
-	this.livechatRoom = new ReactiveVar([]);
+	this.livechatRooms = new ReactiveVar([]);
 	this.selectedAgents = new ReactiveVar([]);
 	this.customFilters = new ReactiveVar([]);
 	this.customFields = new ReactiveVar([]);
 	this.tagFilters = new ReactiveVar([]);
 	this.departments = new ReactiveVar([]);
 
+	const mountArrayQueryParameters = (label, items, index) => items.reduce((acc, item) => {
+		const isTheLastElement = index === items.length - 1;
+		acc += `${ label }[]=${ item }${ isTheLastElement ? '' : '&' }`;
+		return acc;
+	}, '');
+
+	const mountUrlWithParams = (filter, offset) => {
+		const { status, agents, department, from, to, tags, customFields, name: roomName } = filter;
+		let url = `livechat/rooms?count=${ ROOMS_COUNT }&offset=${ offset }`;
```

Comment:
There is a problem here.
On the old publication, we were returning the `department` object of each room, the `department` name is displayed on the chats list.
After implementing the REST endpoint, the `department` name is no longer displayed, because the endpoint doesn't return this object anymore.
Since we need this information to keep the same feature working, you'll probably need to make some changes on the `rooms` endpoint.